fit_meet/settings.py

```diff
@@ -43,6 +40,7 @@
     'category.apps.CategoryConfig',
     'category_location.apps.CategoryLocationConfig',
     'poll.apps.PollConfig',
+    'teams.apps.TeamsConfig'
```

Comment:
Commas are allowed also on the last element. So, it's safer to add a comma to avoid creating issues to other PRs
```suggestion
    'teams.apps.TeamsConfig',
```

Reply:
Yes I understand. I made the change.